### ISSUE TITLE
grouping needed for multiple endocrine neoplasias

### ISSUE THREAD

**Issue by cmungall** [DiseaseOntology/HumanDiseaseOntology#199]
```
  is_a DOID:0050736 ! autosomal dominant disease
      is_a DOID:0050430 ! multiple endocrine neoplasia type 2A ***
      is_a DOID:0080137 ! multiple endocrine neoplasia type 4 ***
      is_a DOID:10016 ! multiple endocrine neoplasia type 2B ***
      is_a DOID:10017 ! multiple endocrine neoplasia type 1 ***
```

consider getting cancer classifications more directly from NCIT:

```
       is_a NCIT:C6432 ! Multiple Endocrine Neoplasia *** 
        is_a NCIT:C3225 ! Multiple Endocrine Neoplasia Type 1
        is_a NCIT:C3226 ! Multiple Endocrine Neoplasia Type 2A
        is_a NCIT:C3227 ! Multiple Endocrine Neoplasia Type 2B
```

**Comment by lschriml** [comment]
DO's subtypes are defined based on OMIM, not NCIt. 
When creating DOIDs, we consult multiple sources.

**Comment by cmungall** [comment]
I'm sorry, I didn't explain myself very well. I am requesting groupings/classifications, not leaf nodes (NCIt subtypes are an important issue, but not one that is central to the original intent of this ticket).

One of the reasons to have an ontology is to group similar things. The 4 DOIDs I listed are all typically considered subtypes of a grouping "Multiple Endocrine Neoplasia" (see for example https://ghr.nlm.nih.gov/condition/multiple-endocrine-neoplasia). I offered the NCIT classification as an example of a grouping class, but the main point is there should be a grouping.

Am I to understand the DOID has no plans for grouping in scenarios like this?
